pkg/plugin/trans.go

```diff
@@ -221,6 +221,17 @@ func (r resourceTransition[T]) handleDeleted(currentlyMigrating bool) (verdict s
 	return verdict
 }
 
+func (r resourceTransition[T]) handleNonAutoscalingUsageChange(newUsage T) (verdict string) {
+	diff := newUsage - r.node.Reserved
+	r.pod.Reserved += diff
+	r.node.Reserved = newUsage
```

Comment:
hm, this is also incorrect, similar-ish to before. As written, `newUsage` corresponds to the _pod's_ usage.